var/spack/repos/builtin/packages/amrvis/package.py

```diff
@@ -0,0 +1,239 @@
+        filter_file(r'^.*\b(usr|opt)\b.*$',
+                    '# Spack removed INCLUDE_LOCATIONS and LIBRARY_LOCATIONS',
+                    'GNUmakefile')
+
+        # Read GNUmakefile into array
+        with open('GNUmakefile', 'r') as file:
+            contents = file.readlines()
+
+        # Edit GNUmakefile INCLUDES and LIBRARIES to use Spack dependencies.
+        # Assuming the default GNUmakefile doesn't change, this is the best
+        # place for LIBRARY_LOCATIONS and INCLUDE_LOCATIONS.
+        line_offset = 63
+        contents.insert(
+            line_offset + 1,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libsm'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 2,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libsm'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 3,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libice'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 4,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libice'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 5,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libxpm'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 6,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libxpm'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 7,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libx11'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 8,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libx11'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 9,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libxt'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 10,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libxt'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 11,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['libxext'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 12,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['libxext'].prefix.include)
+        )
+        contents.insert(
+            line_offset + 13,
+            'LIBRARY_LOCATIONS += {0}\n'.format(spec['motif'].prefix.lib)
+        )
+        contents.insert(
+            line_offset + 14,
+            'INCLUDE_LOCATIONS += {0}\n'.format(spec['motif'].prefix.include)
+        )
+
+        # Write GNUmakefile
+        with open('GNUmakefile', 'w') as file:
+            file.writelines(contents)
+
+    def setup_environment(self, build_env, run_env):
+        if '+mpi' in self.spec:
+            # Set MPI location
+            build_env.set('MPI_HOME', self.spec['mpi'].prefix)
+            build_env.set('MPIHOME', self.spec['mpi'].prefix)
```

Comment:
It uses both??

Reply:
I'm not sure it uses either of them, but I just see both floating around in the AMReX makefiles.

Reply:
🤦‍♂️

Reply:
I see where it sets `MPIHOME=MPI_HOME` now, so it just uses `MPI_HOME`.